platform/src/error_code.rs

```diff
@@ -7,7 +7,7 @@ use core::{convert::TryFrom, fmt, mem::transmute};
 /// 
 /// [error-codes]: https://github.com/tock/tock/blob/master/doc/reference/trd104-syscalls.md#33-error-codes
 #[derive(Clone, Copy, PartialEq, Eq)]
-#[repr(u16)]  // To facilitate use with transmute() in CommandReturn
+#[repr(u32)]
```

Comment:
There should be a comment explaining `#[repr(u32)]`. It is still necessary for the `transmute()` to work, and you should probably document that using `u32` rather than `u16` produces significant code size savings.

Reply:
Done.